actions.go

```diff
@@ -104,3 +112,96 @@ func setLogLevel(loglevel string) {
 		logrus.SetLevel(logrus.InfoLevel)
 	}
 }
+
+type actionCode struct {
+	service  *model.Service
+	isDeploy bool
+}
```

Comment:
This should be in the `model` package. Each field must be exported with the right json annotations